userd/autostart.go

```diff
@@ -151,6 +153,22 @@ func (f failedAutostartError) Error() string {
 	return out.String()
 }
 
+func makeStdStreams(identifier string) (stdout *os.File, stderr *os.File) {
+	var err error
+
+	stdout, err = systemd.NewJournalStreamFile(identifier, syslog.LOG_INFO, false)
+	if err != nil {
```

Comment:
Yep, makes sense, we should have reasonable fallback here.

Reply:
It seems we also want to log the error to stdout so that we learn about possible issues.